01_fib/jimbotech/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.vscode/*
```

Comment:
Only source files should be committed.

Reply:
I usually check in the .gitignore and others also consider it normal practise as per https://stackoverflow.com/questions/5765645/should-you-commit-gitignore-into-the-git-repos, however in this context it might not add a lot, so I removed it.